Driver/Driver/Helper/AppEnvironment.swift

```diff
@@ -15,9 +15,9 @@ enum AppEnvironment: String {
         // FIXME: Change after servis deploy
         switch self {
         case .development:
-            return "https://development.base.url/"
+            return "http://ec2-18-159-211-214.eu-central-1.compute.amazonaws.com/"
         case .production:
-            return "https://production.base.url/"
+            return "http://ec2-18-159-211-214.eu-central-1.compute.amazonaws.com/"
         }
```

Comment:
We should add domain restriction for this url. Otherwise, we shouldn't request anywhere using http.

Reply:
I added true value for allows arbitrary loads key in info.plist so It works but if there is another restriction etc. I can change it. I won't take any action for this comment for now.